notebooks/Perception_Statistics.ipynb

```diff
@@ -479,8 +479,116 @@
     "    inst_caps = cap.filter(def_id=inst_seg_def_id)\n",
     "    interact(draw_with_instance_segmentation, index=(0, inst_caps.shape[0]), opacity=(0.0, 1.0))\n",
     "except DefinitionIDError:\n",
-    "    print(\"No instance segmentation images found\")\n"
+    "    print(\"No instance segmentation images found\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Keypoints\n",
+    "In this section we render the keypoint labeled data for the captured frame."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "from PIL import ImageDraw\n",
```

Comment:
This is a lot of code for a user-facing notebook. Most of this code should be moved into the visualization code in datset-insights so that it can be reused by users in other notebooks.

Reply:
Agreed, moved most of the code to its own file